atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTopologyValidatorTest.java

```diff
@@ -518,11 +523,27 @@ private static CassandraServer createCassandraServer(String hostname) {
         return CassandraServer.of(hostname, mock(InetSocketAddress.class));
     }
 
-    private static void setHostIds(Collection<CassandraClientPoolingContainer> containers, HostIdResult... hostIds) {
+    private void setHostIds(Collection<CassandraClientPoolingContainer> containers, HostIdResult... hostIds) {
+        TApplicationException softFailureException = new TApplicationException(TApplicationException.UNKNOWN_METHOD);
```

Comment:
Doing it this way required the least changes, since the alternative is... chunky. Depends on the answer to the internal client q too if we even need this, but otherwise lmk if you prefer pushing this up to each individual test.